projects/uswds-components/src/lib/sidenav/sidenav.component.html

```diff
@@ -6,28 +6,7 @@
 </ul>
 
 <ng-template #sideNavLinkTemplate let-link>
-  <ng-container [ngSwitch]="link.mode">
```

Comment:
Extracting out this navigation mode behavior into a separate component for re-use in breadcrumbs